gcloud/bigquery/client.py

```diff
@@ -133,3 +134,22 @@ def copy_table(self, name, destination, *sources):
         :returns: a new ``CopyJob`` instance
         """
         return CopyJob(name, destination, sources, client=self)
+
+    def extract_table_to_storage(self, name, source, *destination_uris):
+        """Construct a job for copying one or more tables into another table.
```

Comment:
"Into another table" is the `copy_table` one -- this one extracts the data (as CSV, newline-delmited JSON, or one other weird format) to storage blob(s).